recipe/meta.yaml

```diff
@@ -89,7 +92,7 @@ outputs:
       run:
         - python
         - {{ pin_subpackage(name, min_pin='x.x', max_pin='x.x') }}
```

Comment:
```suggestion
        - {{ pin_subpackage({{ name }}, min_pin='x.x', max_pin='x.x') }}
```
Still learning about multi-output recipes, but should this `name` be the variable `{{ name }}`?

Reply:
It's already in a jinja context, as argument of jinja function pin_subpackage.

Reply:
Definitely only saw parentheses, sorry!